data/games/guilty_gear_xrd_rev_2/movespecs/answer.json

```diff
@@ -123,6 +123,49 @@
         "imageId": "j_D"
       }
     ],
+    [
+      {
+        "name": "Ground Throw",
+        "inputs": [
+          "4H",
+          "6H"
+        ],
+        "imageId": "Ground_Throw",
+        "conditions": "While next to the opponent"
+      },
+      {
+        "name": "Air Throw",
+        "inputs": [
+          "j.4H",
+          "j.6H"
+        ],
+        "imageId": "Air_Throw",
+        "conditions": "While next to the opponent"
```

Comment:
Would it make more sense to use `c.` with throws rather than specifying the same information with a condition?

Reply:
I think the ranges required for f.S to change into c.S is different from where 6H becomes a throw. I'd need to test that to be certain, though.

Reply:
I tried it out and it looks like close range and throw range differ. Close range is well, close. Throw range is essentially no space in-between the two characters.